libs/common/ui/src/components/GeneralAutocomplete.tsx

```diff
@@ -235,6 +235,7 @@ export function GeneralAutocompleteMulti<T extends string>({
       filterOptions={(options, { inputValue }) =>
         options.filter(
           (opt) =>
+            opt.key.toLowerCase().includes(inputValue.toLowerCase()) ||
             opt.label.toLowerCase().includes(inputValue.toLowerCase()) ||
             opt.alternateNames?.some((name) =>
```

Comment:
These alternate names should already have the real name, when silly mode is enabled

Reply:
https://github.com/search?q=repo%3Afrzyc%2Fgenshin-optimizer%20alternateNames&type=code

For example: 
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/f02d92ab-94cf-42d9-b733-bb82231feb47)

Reply:
Done. I guess what you were trying to say is that traveler name is not included in their key

<img width="324" alt="image" src="https://github.com/frzyc/genshin-optimizer/assets/44922829/18d6d179-900a-4244-ac71-81254e7b4e97">

Reply:
This way also supports translated names